window/ui/src/main.mjs

```diff
@@ -0,0 +1,21 @@
+// import { createRoot } from "../../../node_modules/react-dom/client";
+// import { createElement as h } from "../../../node_modules/react";
+
+// import App from './app.mjs';
+
+// const root = createRoot(document.getElementById("app"));
+// root.render(h(App));
+
+// NOTE - frontend setup sucks, I dropped it because setting up build is outside of scope
```

Comment:
I gave up on frontend framework install because it was getting annoying. Seems like a task more for another PR